routes/users/users.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express')
+const users = express.Router()
+
+let usersArr = [{
+    username: "user1234",
+    email: "user1234@gmail.com",
+    activationDate: "January 1, 1970"
+},
+{
+    username: "Sam",
+    email: "Sam4Seven@gmail.com",
+    activationDate: "January 20, 2020"
+}]
+
+users.get("/", (req, res) => {
+    res.json(usersArr)
+})
+
+users.get("/:id", (req, res) => {
+    res.json(usersArr[req.params.id])
+})
+
+users.post("/", (req, res) => {
+    console.log(req)
+    usersArr.push(req.body)
+    res.json([usersArr, req.body])
+})
+
+users.delete("/:id", (req, res) => {
```

Comment:
Technically your users don't have id's - you didn't give them any :) You are correct in your actions here, but ideally you'd give each user an id that's always constant. Otherwise you delete user with id 4 and so user with id 5 now has an id of 4. And that's not the best practice.